andela/Titan-Devs
# 163518693 User Follow Each Other

#### What does this PR do?
Enable user to follow auther
#### Description of Task to be completed?
```POST /api/v1/profiles/:username/follow```
```DELETE /api/v1/profiles/:username/follow```
```GET /api/v1/profiles/:username/followers```
```GET /api/v1/profiles/:username/followings```
#### How should this be manually tested?
-Clone the repo and checkout `ft-user-follow-each-other-163518693` branch
- Using any `API client tool(usually POSTMAN)`,
- Create the user and login to get the `access-token`
- Make request to the above end point with existing ```username```
- Make sure to pass the token under `request headers`. It should be {`Authorization`: `Bearer <access-token>`} KEY, VALUE PAIR
#### Any background context you want to provide?
before running anything, ensure packages are updated ```yarn``` and no pending migration ```sequelize db:migrate```
#### What are the relevant pivotal tracker stories?
163518693
